### ISSUE TITLE
Autohotkey Rune Shooter

### ISSUE THREAD

**Issue by scoxdomain** [Mytherin/Tibialyzer#259]
Hello!

Can you please teach me how to throw AoE Runes (Thunderstorm, Avalanches, etc) using Middle Mouse button?
I don´t know how to input in Autohotkeys screen. 

Thanks.

**Comment by fewufu** [comment]
~MButton::3

Would press hotkey 3 on middle mouse click.

**Comment by Mackbit** [comment]
If you want to instantly shoot an AoE rune at the mouse location with crosshairs, by using the middle mouse button, you can use the following:

`~MButton::`
` Send {F1}`
`Sleep, 50`
`MouseClick, left`
`return`

where {F1} is the hotkey for your rune, which can be changed to something else (you can put any hotkey inside the {} brackets. If you want to use a combination, for instance shift + something, or ctrl + something, you can put a ^ sign before the brackets for ctrl, and a + sign for shift as in the case below:

` Send +{F1}`
(This will send shift + F1)
` Send ^{F1}`
(This will send ctrl + F1)

**Comment by Mytherin** [comment]
I'm pretty sure that usage of AutoHotkey is against the Tibia TOS. One button should map to one action. Having a delay and a separate action is similar to botting.

**Comment by Mackbit** [comment]
@Mytherin: Yes, I am also pretty sure it is against the Tibia TOS. It should only be used at your own risk.

**Comment by scoxdomain** [comment]
Thank you guys, you've helped me a lot.
